trpc/router/projectV2.ts

```diff
@@ -82,6 +82,10 @@ export const project = router({
         try {
             const projectIds = await getStarredProjectsIds(activityId).execute();
 
+            if (!projectIds?.length) {
+                return null;
```

Comment:
why not []?

Reply:
For simplify render condition

Reply:
You simplify one line to mess up with nullable types :(